pkg/utils/host.go

```diff
@@ -0,0 +1,134 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/golang/glog"
+
+	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
+)
+
+const (
+	SriovConfBasePath     = "/etc/sriov-operator"
+	PfAppliedConfig       = SriovConfBasePath + "/pci"
+	HostSriovConfBasePath = "/host" + SriovConfBasePath
+	HostPfAppliedConfig   = HostSriovConfBasePath + "/pci"
+)
+
+type StoreManagerInterface interface {
+	ClearPCIAddressFolder() error
+	SaveLastPfAppliedStatus(pciAddress string, PfInfo *sriovnetworkv1.Interface) error
+	LoadPfsStatus(pciAddress string, chroot bool) (*sriovnetworkv1.Interface, bool, error)
+}
+
+type StoreManager struct {
+}
+
+// NewStoreManager: create the initial folders needed to store the info about the PF
```

Comment:
Why don't we store SriovNetworkNodeState object here like we did for systemd service?

Reply:
the systemd file is different.
here we have a per PF info of the latest applied policy so we know if we need to reset the device or not